src/plugins/orangehrmRecruitmentPlugin/Dao/VacancyDao.php

```diff
@@ -211,4 +203,27 @@ public function getPublishedVacancyList(): array
         $qb->addOrderBy('vacancy.updatedTime', ListSorter::DESCENDING);
         return $qb->getQuery()->execute();
     }
+
+    /**
+     * @param int $jobTitleId
+     * @return JobTitle|null
+     */
+    public function isActiveJobVacancy(int $jobTitleId): ?JobTitle
```

Comment:
```suggestion
    public function isActiveJobTitle(int $jobTitleId): ?JobTitle
```
return type also wrong with the method name. if you want to get job title also (not a boolean) use `JobTitleService`

Reply:
fixed